integration_tests/kpi.feature

```diff
@@ -0,0 +1,26 @@
+Feature: KPI integration tests
+
+  Scenario: House is not self sufficient
+     Given we have a scenario named kpi.house_not_self_sufficient
+     And d3a is installed
+     When we run the d3a simulation on console with kpi.house_not_self_sufficient for 24 hrs
+     Then the export functionality of kpi of {'Grid': 1.0, 'House 1': 0.0} are correctly reported
+
+  Scenario: House is fully self sufficient
+     Given we have a scenario named kpi.house_fully_self_sufficient
+     And d3a is installed
+     When we run the d3a simulation on console with kpi.house_fully_self_sufficient for 24 hrs
+     Then the export functionality of kpi of {'Grid': 1.0, 'House 1': 1.0} are correctly reported
+
+  Scenario: House is partially self sufficient
+     Given we have a scenario named kpi.house_partially_self_sufficient
+     And d3a is installed
+     When we run the d3a simulation on console with kpi.house_partially_self_sufficient for 24 hrs
+     Then the export functionality of kpi of {'Grid': 1.0, 'House 1': 0.34925} are correctly reported
+
+
+  Scenario: House fully self_sufficiency with storage
+     Given we have a scenario named kpi.house_self_sufficient_with_ess
+     And d3a is installed
+     When we run the d3a simulation on console with kpi.house_self_sufficient_with_ess for 24 hrs
+     Then the export functionality of kpi of {'Grid': 1.0, 'House 1': 1.0} are correctly reported
```

Comment:
Nit picking, but only the SS of the House is changing, so I suggest to only parse this value.

Reply:
Yes, but in future there could be more integration tests checking `self_sufficiency` of multiple areas al together.